sunspec/__init__.py

```diff
@@ -1,5 +1,5 @@
 
 # pysunspec version
-version = '1.0.8'
+version = '2.0.0'
```

Comment:
Interestingly doesn't match the version in their setup.py?

Reply:
Looks like the both versions were updated to 2.0.0 at the same time and since then only setup.py has been kept up to date.

Reply:
Presumably that doesn't matter to us